molecule/default/tasks/taint.yml

```diff
@@ -227,6 +227,10 @@
       that:
       - _result.changed
 
+  - name: Display all taints
+    debug:
+      msg: "{{ taint_patch_1 }} + {{ taint_patch_2 }}"
+
```

Comment:
Can you revert this change?

Reply:
I do this to debug the CI which was no stable
Will remove